server/routes/api/ads.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const router = express.Router();
```

Comment:
switch lines 2 + 3, requires/imports come first. You can add also 1 empty line 😃